chalicelib/checks/wrangler_checks.py

```diff
@@ -488,13 +489,21 @@ def identify_files_without_filesize(connection, **kwargs):
                 'upload_key': hit.get('upload_key')
             }
             problem_files.append(hit_dict)
+    check.brief_output = '{} files with no file size'.format(len(problem_files))
     check.full_output = problem_files
     if problem_files:
         check.status = 'WARN'
         check.summary = 'File metadata found without file_size'
-        check.description = "One or more files that are released/released to project/uploaded don't have file_size."
+        status_str = 'released/released to project/uploaded'
```

Comment:
Would we want the status_str to update based on status option? If not then you might still want to add pre-release here.

Reply:
Oh nevermind i missed the statement below before. But the second part still stands.